mantidproject/mantid
# Separate dialogs and widgets into different mantidqt subpackages

**Description of work.**

The `PlotSelectionDialog` class was poorly named and can be confused with the new `PlotSelectionWidget` name. This work creates a new `mantidqt.dialogs` subpackage that contains only Qt `Dialog` types and not general widgets. The `PlotSelectionDialog` dialog has been renamed to ``

This is purely a reoganization of code and no functionality has changed.

**To test:**

Build the workbench, check it starts and you can plot something.

*No associated issue*

Does this update require release notes?


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
